test/refactor_nrepl/ns/libspec_allowlist_test.clj

```diff
@@ -1,8 +1,13 @@
 (ns refactor-nrepl.ns.libspec-allowlist-test
+  {:clj-kondo/config '{:linters {:unused-namespace {:exclude [something.important
```

Comment:
Let's rename `something.important` to `from-ns-attr-map` (just like in my previous comment)